doc/source/tools/pointwise/pointwise.rst

```diff
@@ -0,0 +1,335 @@
+
+.. image:: images/mesh_file.png
+    :align: center
+    :width: 400px
+
+
+.. _moving_3d: 
+
+--------------------------
+Moving to 3D
+--------------------------
+
+This section will give you an idea of the tools that can be used to create the following 3D mesh.
+
+.. image:: images/final_mesh_3d2.png
+    :align: center
+
+----
+
+Extrusions
+-----------
+
+The best way to create 3D **block** meshes from 2D geometries is by using extrusions. The following steps can be achieved from the final 2D geometry generated previously. 
+
+- **Set Dimension** to 3D in the **CAE** menu
+- Select all the geometry and click on **Translate** in the **Edit** and **Transform** menu.
+- Insert **0.5 0 0** in the **Offset Vector** box and press enter. Click on **ok** to leave the menu. 
+
+.. image:: images/translation.png
+    :align: center
+
+The last action should have translated the geometry from 0.5 in the x direction. This translation is needed to avoid the creation of tetrahedral cells in the center of the mesh during the rotational extrusion. 
+
+- In the **Create** menu, select **Extrude** and **Rotate**
+- click **Done** after the first menu appears. Enter the desired number of cells in the **Steps** box. 50 is used in this example.
+- Insert 360 in the **Angle** box.
+- Select the **Point and Direction** option. Insert the point **0 0 0** in the **XYZ** box and choose the Y direction.
+- Click the **Run** button on the top.  
+
+.. image:: images/extrusion.png
+    :align: center
+
+The following mesh should be obtained at the end of these steps.
+
+.. image:: images/final_mesh_3d.png
+    :align: center
+
+----
+
+filling the center
+--------------------
+
+An initial translation makes sure no tetrahedral is created at the center of the mesh. The interior of the inlet must however be filled.
+The next step will guarantee the hole created is filled.
+
+- Split the 3 inner round connectors into 4 parts using the **Split** tool in the **Edit** menu. These connectors are distributed along the y-axis and should have a diameter of 1. 
+
+.. image:: images/split_3d.png
+    :align: center
+
+- Generate a square on the bottom of the hole of the geometry with the **2 Point Curve**. The four coordinates of the square are (0.25, 0, 0), (-0.25, 0, 0), (0, 0, 0.25) and (0, 0, -0.25)
+- Connect the edges of the square to the lowest points that appeared after the splitting of the inside connectors.
+
+The image below represents the current state of the geometry. 
+
+.. image:: images/filling1.png
+    :align: center
+
+At this point, there is a problem in the geometry. The previous separation created connectors of uneven dimensions. Hovering the mouse over the inside connectors quickly shows in the bottom right corner that some are of size 14 while some are of size 13. However, the opposing sides need to be equal in a structured mesh. Some inside, middle and outside connectors will need to be modified.
```

Comment:
Maybe add information or labels in the figure about inside,middle and outside? I was no sure what part of the geometry you are referring to.